doc/2/guides/develop-on-kuzzle/1-embedded-sdk/index.md

```diff
@@ -0,0 +1,164 @@
+The behavior of the [realtime:subscribe](/sdk/js/7/controllers/realtime) method is slightly different when it's used with the Embedded SDK.  
+Learn more about [Backend Realtime Subscriptions](/core/2/guides/develop-on-kuzzle/1-embedded-sdk#backend-realtime-subscriptions)
+:::
+
+**Example:** _Create a new document by using the [document.create](/sdk/js/7/controllers/document/create) method_
+```js
+// After application startup
+
+// [...]
+
+// Creates a document
+await app.sdk.document.create('nyc-open-data', 'yellow-taxi', {
+  name: 'Aschen',
+  age: 27
+})
+```
+
+## Query method
+
+<!-- Duplicate /core/2/guides/getting-started/6-write-application -->
+
+The low level [query](/sdk/js/7/core-classes/kuzzle/query) method can also be used to **send custom requests to the Kuzzle API**.  
+
+**Example:** _Execute a custom controller action with the [query](/sdk/js/7/core-classes/kuzzle/query) method_
+```js
+// After application startup
+
+// [...]
+
+// Execute a custom controller action
+await app.sdk.query({
+  controller: 'greeting',
+  action: 'sayHello',
+  name: 'Aschen'
+})
+```
+
+## User impersonation
+
+By default, when using the Embedded SDK, requests **don't have the same context as the original request** received by Kuzzle.
+
+Typically, the `request.context.user` property is not set and thus **[Kuzzle metadata](/core/2/some-link) will not be set when creating or updating documents**.
+
+It is possible to use the same user context as the original request with the Embedded SDK, for this purpose it is necessary to use the [EmbeddedSDK.as](/core/2/some-link) method.
+
+**Example:** _Creating a document as the original API user to preserve Kuzzle metadata_
+```js
+app.controller.register('drivers', {
+  actions: {
+    create: {
+      handler: async request => {
+        const originalUser = request.context.user
+
+        return app.sdk.as(originalUser).document.create(
+          'nyc-open-data',
+          'yellow-taxi',
+          { name: 'Aschen' })
+       }
+    }
+  }
+})
+```
+
+::: warning
+User permissions are applied only once, when a request is received by Kuzzle through the exposed API.  
+If a request is authorized, then all subsequent calls to the API performed with [EmbeddedSDK.as](/core/2/some-link) are always authorized, even if they are made to execute API actions that a user is normally forbidden from.
+:::
+
+## Backend Realtime Subscriptions
+
+Realtime subscriptions should be made using the [Realtime Controller](/sdk/js/7/controllers/realtime) **just after the application has started**.  
+
+Realtime subscriptions performed by an application are used so that an application gets notified about changes, and can act upon them. The behavior is much the same than when a client subscribes, but since the entity performing the subscription is different (client vs. application), the feature accessible by an application has some new options to fine tune how notifications are propagated across a Kuzzle cluster.
```

Comment:
```suggestion
Realtime subscriptions performed by an application are used so that an application gets notified about changes, and can act upon them. The behavior is the same as when a client subscribes, but since the entity performing the subscription is different (client vs. application), the feature accessible by an application has some new options to fine tune how notifications are propagated across a Kuzzle cluster.
```